packages/commonwealth/client/scripts/views/components/component_kit/cw_toggle.tsx

```diff
@@ -40,7 +41,7 @@ export const CWToggle = (props: ToggleProps) => {
           className,
         },
         ComponentType.Toggle
-      )}
+      ).concat(` ${size}`)}
```

Comment:
please add the `size` to the toggle style props above and just use the getClasses method.

Reply:
> please add the `size` to the toggle style props above and just use the getClasses method.

Done.

Also moved the new design component to new_designs folder:

![image](https://github.com/hicommonwealth/commonwealth/assets/30223098/06cbb5e4-3496-4bef-afe1-27e9f1e9a169)